motr/m0crate/crate_index.c

```diff
@@ -958,13 +1099,23 @@ static int cr_idx_w_execute(struct cr_idx_w *w,
 		keys[i].f_container = w->key_prefix.f_container;
 	}
 
-	rc = op->fill_kv(w, keys, &kv, nr_keys);
+	/* populating key prefix */
+	for(i = 0; i < kpart_one_size; i++) {
+		kpart_one[i] = 'A';
+	}
```

Comment:
memset() will be better choice for this?
```
memset(kpart_one, 'A', kpart_one_size);
```

Reply:
Fixed.